### PR TITLE
Add features

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: Drasko DRASKOVIC <drasko.draskovic@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ For more details, check out the [official documentation][docs].
 
 - Protocol bridging (i.e. HTTP, MQTT, WebSocket, CoAP)
 - Device management and provisioning
-- Fine-grained access control
+- Data persistence - linearly scalable high-performance [Cassandra][cassandra] cluster
```
Comment (mijicd) [inline README.md]: Perhaps `linearly scalable [data storage][cassandra]` might be better.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ For more details, check out the [official documentation][docs].
...
+- Security - user, device and application authentication and authorization via access control
```
Comment (mijicd) [inline README.md]: I'd prefer the old statement: `Fine-grained access control`.

File: README.md [hunk 3]
```diff
@@ -19,7 +19,9 @@ For more details, check out the [official documentation][docs].
...
+- Service logging and monitoring
```
Comment (mijicd) [inline README.md]: Perhaps this would be better: `Platform logging and instrumentation support`

**Comment by drasko** [comment]
Finally I am afraid that this can introduce clutter, as you said. I do not mind dropping PR.

**Review by mijicd — APPROVED** [review]

**Pull request merged**